test/access-token.test.js

```diff
@@ -388,7 +388,7 @@ describe('AccessToken', function() {
       var Token = givenLocalTokenModel();
 
       // Overwrite User settings - enable eternal tokens
-      Token.app.models.User.settings.allowEternalTokens = true;
+      app.registry.getModelByType(loopback.User).settings.allowEternalTokens = true;
```

Comment:
there's more to it: the other way around with the initial formula is not working. what do we do?